packages/mds-agency/request-handlers.ts

```diff
@@ -330,16 +320,32 @@ export const submitVehicleEvent = async (req: AgencyApiRequest, res: AgencyApiRe
 
     // database write is crucial; failures of cache/stream should be noted and repaired
     const recorded_event = await db.writeEvent(event)
+    const { telemetry } = recorded_event
+
+    if (telemetry) {
+      await db.writeTelemetry(Array.isArray(telemetry) ? telemetry : [telemetry])
```

Comment:
Could use `normalizeToArray()`

Reply:
I forgot we had that now! Did you happen to rip that out into a function in `mds-utils` yet?

Reply:
Yeah I have a branch for it